disdrodb/l0/check_configs.py

```diff
@@ -69,48 +289,77 @@ def check_bin_consistency(sensor_name: str) -> None:
     np.testing.assert_allclose(velocity_bin_upper - velocity_bin_width / 2, velocity_bin_center)
 
 
-def check_variable_keys_consistency(sensor_name: str) -> None:
-    """Check attributes consistency from config file.
+def get_chunksizes(sensor_name: str, file_name: str) -> list:
+    """get chunksizes from config file.
```

Comment:
This are not chunksizes !  These are the disdrometer measurement bins. 
`(center, bounds, width)` characterize the drop (diameter and velocity) measurement bins of the disdrometer ! 
Chunksizes (in the netcdf encodings) specify just the size of blocks of data to be saved into a netCDF (it's purely a compression argument) !